### DISCUSSION TITLE
v4.6.3 api/v1/search return empty data

### DISCUSSION THREAD

**Discussion by nancheal** [DependencyTrack/dependency-track#2167] [General]
with same searchtext like: xxx
api/v1/search empty
curl -X GET "http://127.0.0.01:8081/api/v1/search?query=xxx" -H  "accept: application/json" -H  "X-Api-Key: xxxx"
![image](https://user-images.githubusercontent.com/12165569/202690369-e878eef7-e3c9-4a29-ae83-be1b8951ddd6.png)
api/v1/project not empty
![image](https://user-images.githubusercontent.com/12165569/202690835-4efcfea4-bf8e-4b11-a757-1c77021aa62f.png)
is this a bug of api/v1/search?

**Comment by syalioune** [comment]
Hello @nancheal 
Which version of DT do you use ?
This is most likely due to bug https://github.com/DependencyTrack/dependency-track/issues/2104 introduced in v4.6.0
If so a fix will be provided in the next 4.7.0 release.

**Comment by nancheal** [comment]
thanks for replay
i use v4.6.3 and "Enable fuzzy CPE matching" 
Might be the same problem as you said
when will 4.7.0 be released?

**Reply by syalioune** [reply]
There are no set date ! You can check the progress using https://github.com/DependencyTrack/dependency-track/milestone/22
